install.sh

```diff
@@ -134,8 +146,8 @@ sudo chown "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
 
 # Delete ${CREW_PREFIX}/{var,local} symlinks on some Chromium OS distro
 # if they exist.
-[ -L ${CREW_PREFIX}/var ] && sudo rm -f "${CREW_PREFIX}/var"
-[ -L ${CREW_PREFIX}/local ] && sudo rm -f "${CREW_PREFIX}/local"
+[ -L "${CREW_PREFIX}"/var ] && sudo rm -f "${CREW_PREFIX}/var"
+[ -L "${CREW_PREFIX}"/local ] && sudo rm -f "${CREW_PREFIX}/local"
```

Comment:
Quoting the whole string is cleaner (imho).

Reply:
making that change...